misc/opentelemetry/mzcompose.py

```diff
@@ -18,7 +18,8 @@
         "jaeger-all-in-one",
         {
             "image": "jaegertracing/all-in-one:latest",
-            "ports": [16686, 14268, 14250],
+            "ports": ["16686:16686", 14268, 14250],
+            # "allow_host_ports": True,
```

Comment:
Don't you want this on so that 16686 goes through??

Reply:
need to push the most recent version!